code/modules/vehicles/cars/vim.dm

```diff
@@ -71,6 +82,10 @@
 	if(atom_integrity == max_integrity)
 		SEND_SOUND(newoccupant, sound('sound/mecha/nominal.ogg',volume=50))
 
+/obj/vehicle/sealed/car/vim/mob_try_exit(mob/M, mob/user, silent = FALSE, randomstep = FALSE)
```

Comment:
Choose a better name than `M`, please!

Reply:
my bad, i just copied it over from the "/obj/vehicle/sealed/car/" file without changing the variable name

Reply:
done